examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/README.md

```diff
@@ -1,179 +1,96 @@
-Step-by-Step
-============
-This document describes the step-by-step instructions to run large language models (LLMs) on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with PyTorch and Intel® Extension for PyTorch.
+Weight-only quantization
+===============
 
-The script `run_clm_no_trainer.py` supports `GPTJ`, `OPT`, `LLaMA2`, `BLOOM` and `Falcon` quantization and validates last word prediction accuracy with [lm_eval](https://github.com/EleutherAI/lm-evaluation-harness.git) now, and we are adding more models.
-
-# Prerequisite
-## 1. Create Environment
+##  Prerequisite
 ```
 # Installation
 pip install -r requirements.txt
 ```
 
-# Run
+## Support status on HPU
 
-Here is how to run the scripts:
+Below is the current support status on Intel Gaudi 2 AI Accelerator with PyTorch.
 
-**Causal Language Modeling (CLM)**
+| woq_algo |   Status  |
+|--------------|----------|
+|   GPTQ   |  &#10004;|
 
-`run_clm_no_trainer.py` quantizes the large language models using the dataset [NeelNanda/pile-10k](https://huggingface.co/datasets/NeelNanda/pile-10k) calibration and validates `lambada_openai`, `piqa`, `winogrande`, `hellaswag` and other datasets accuracy provided by lm_eval, an example command is as follows.
-### GPT-J-6b
+> Validated models: [`meta-llama/Llama-2-7b-hf`, `EleutherAI/gpt-j-6B`, `facebook/opt-125m`]
 
-#### Quantization
+## Support status on CPU
 
-```bash
-# "--woq_algo GPTQ" is used to enable GPTQ algorithms
-# "--double_quant_type BNB_NF4" is used to enable double quant algorithms
-python run_clm_no_trainer.py \
-    --model EleutherAI/gpt-j-6B \
-    --dataset NeelNanda/pile-10k \
-    --quantize \
-    --woq_algo GPTQ \
-    --woq_bits 4 \
-    --woq_scheme asym \
-    --woq_group_size 128 \
-    --gptq_max_seq_length 2048 \
-    --gptq_use_max_length \
-    --double_quant_type "BNB_NF4" \
-    --output_dir saved_results
+Below is the current support status on 4th Gen Intel® Xeon® Scalable Processor (codenamed Sapphire Rapids) with PyTorch.
```

Comment:
same, no need to call out 4th Gen (codenamed Sapphire Rapids),  just Intel® Xeon® Scalable Processor